apps/api/src/routers/sidecar.ts

```
@@ -45,6 +46,29 @@ const app = new Hono()
+
+        let subdomain = deployment.project.slug;
+        if (deployment.target === "PREVIEW") {
+          subdomain = `${deployment.project.slug}-${deployment.gitCommitHash.slice(0, 7)}`;
+        }
+        await invalidateCacheByTag(`site:${subdomain}`);
+
+        // {"commitHash":"045ad814b6c07e16c8193418dbbc899e3a3ef257","spa":true,"htmlRoutes":{"/":"index.html"}}
+        // {"commitHash":"d60d2d71a08c9e174213eb34612345acb910ff50","htmlRoutes":{"/404":"404.html","/blog/html-intro":"blog/html-intro/index.html","/blog":"blog/index.html","/":"index.html","/projects":"projects/index.html","/projects/zaggonaut":"projects/zaggonaut/index.html"}}
+        const htmlRoutes: Record<string, string> = {};
+        if (deployment.metadata?.htmlRoutes) {
+          for (const route of deployment.metadata?.htmlRoutes) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)